assets/src/modules/Tooltip.js

```diff
@@ -0,0 +1,197 @@
+        const layerName = layerTooltipCfg.name;
+        const tooltipLayer = this._tooltipLayers.get(layerName);
+        this._displayGeom = layerTooltipCfg.displayGeom;
+
+        // Styles
+        const fill = new Fill({
+            color: 'transparent',
+        });
+
+        const stroke = new Stroke({
+            color: 'rgba(255, 255, 255, 0.01)', // 'transparent' doesn't work for lines. Is it a bug in OL?
+        });
+
+        const hoverColor = layerTooltipCfg.colorGeom;
+
+        const hoverStyle = feature => {
+            if (['Polygon', 'MultiPolygon'].includes(feature.getGeometry().getType())) {
+                return new Style({
+                    fill: fill,
+                    stroke: new Stroke({
+                        color: hoverColor,
+                        width: 3
+                    }),
+                });
+            } else if (['LineString', 'MultiLineString'].includes(feature.getGeometry().getType())) {
+                return new Style({
+                    stroke: new Stroke({
+                        color: hoverColor,
+                        width: 6
+                    }),
+                });
+            } else if (['Point', 'MultiPoint'].includes(feature.getGeometry().getType())) {
+                return new Style({
+                    image: new Circle({
+                        fill: fill,
+                        stroke: new Stroke({
+                            color: hoverColor,
+                            width: 3
+                        }),
+                        radius: 5,
+                    }),
+                    fill: fill,
+                    stroke: new Stroke({
+                        color: hoverColor,
+                        width: 3
+                    }),
+                });
+            }
+            return undefined;
+        }
+
+        if (tooltipLayer) {
+            this._activeTooltipLayer = tooltipLayer;
+        } else {
+            const url = `${lizUrls.service.replace('service','tooltips')}&layerId=${layerTooltipCfg.id}`;
+
+            const vectorStyle = new Style({
+                image: new Circle({
+                    fill: fill,
+                    stroke: stroke,
+                    radius: 5,
+                }),
+                fill: fill,
+                stroke: stroke,
+            });
+
+            this._activeTooltipLayer = new VectorLayer({
+                source: new VectorSource({
+                    url: url,
+                    format: new GeoJSON(),
+                }),
+                style: vectorStyle
+            });
+
+            // Load tooltip layer
+            this._activeTooltipLayer.once('sourceready', () => {
+                mainEventDispatcher.dispatch('tooltip.loaded');
+            });
+
+            this._activeTooltipLayer.on('error', () => {
+                console.log(`Tooltip layer '${layerName}' could not be loaded.`);
```

Comment:
Could be great to display a message to the user with `addMessage` . Better if it can be translated. 
`An error prevented to load the tooltip data`

In the console, try to also display the real error message if possible

Reply:
Done. The real error message from OL don't say anything more.